src/crate/crash/commands.py

```diff
@@ -91,47 +91,82 @@ def __call__(self, cmd, *args, **kwargs):
             cmd._autocomplete and 'ON' or 'OFF'
         )
 
-class ClusterCheckCommand(Command):
-    """ print failed cluster checks """
-
-    CLUSTER_CHECK_STMT = """
-        SELECT description AS "Failed checks"
-        FROM sys.checks
-        WHERE passed = false 
-        ORDER BY id ASC"""
 
-    @noargs_command
-    def __call__(self, cmd, *args, **kwargs):
-        if cmd.connection.lowest_server_version >= StrictVersion("0.52.0"):
-            self.execute(cmd)
-        else:
-            tmpl = 'Crate {version} does not support the cluster "check" command.'
-            cmd.logger.warn(tmpl.format(version=cmd.connection.lowest_server_version))
+class CheckCommand(Command):
 
-    def execute(self, cmd):
-        success = cmd._execute(ClusterCheckCommand.CLUSTER_CHECK_STMT)
+    def execute(self, cmd, stmt, msg):
+        success = cmd._execute(stmt)
         cmd.exit_code = cmd.exit_code or int(not success)
         if not success:
             return False
         cur = cmd.cursor
         print_vars = {
             's': 'S'[cur.rowcount == 1:],
-            'rowcount': cur.rowcount
+            'rowcount': cur.rowcount,
+            'msg': msg
```

Comment:
I wouldn't pass the `msg` variable but define a class variable 
and use `self.CHECK_NAME` here